config/config-api/src/com/thoughtworks/go/config/AdminUser.java

```diff
@@ -35,6 +36,9 @@ public AdminUser(final CaseInsensitiveString name) {
     }
 
     public void validate(ValidationContext validationContext) {
+        if(name == null || StringUtil.isBlank(CaseInsensitiveString.str(name)))
```

Comment:
There's already a `CaseInsensitiveString#isBlank` that you could use instead of pulling out the raw string.